src/Folder.h

```diff
@@ -93,7 +93,6 @@
 -(NSString *)lastUpdateString;
 -(NSString *)username;
 -(NSString *)password;
--(NSDictionary *)attributes;
```

Comment:
Oh! Sorry, I mis-read the dif and thought it was a `+` (I have been cutting onions for dinner :hocho: :sob:)